engine/chassis.ts

```diff
@@ -486,9 +486,9 @@ export class Program {
             bindings.descriptorSets.map((descriptorSet, index) => commandBuffer.bindDescriptorSet(index, descriptorSet));
         } else {
             for (let i = 0; i < bindings.descriptorSets.length; i++) {
-                const offsets = bindings.dynamicOffsets[i];
+                const offsets = [];
```

Comment:
Remember this is a (if not the most) frequently-invoked function, so creating a temp array on every invocation would not be ideal